notanorm/base.py

```diff
@@ -739,13 +751,16 @@ def execute_ddl(self, sql: str, *dialect: str, ignore_existing=True):
     def clear_model_cache(self):
         self.__model_cache = None
 
-    def execute(self, sql: str, parameters=(), _script=False, write=True):
+    def execute(
+        self, sql: str, parameters=(), _script=False, write=True, no_capture=False
+    ):
         if "alter " in sql.lower() or "create " in sql.lower():
             self.__model_cache = None
 
-        self.__debug_sql(sql, parameters)
+        if not no_capture:
```

Comment:
IMO we should definitely log model lookups. Have used that to debug perf issues in the past.

Reply:
removed